examples/pom.xml

```diff
@@ -267,25 +242,25 @@
         <dependency>
             <groupId>jakarta.xml.bind</groupId>
             <artifactId>jakarta.xml.bind-api</artifactId>
-            <version>3.0.0-RC3</version>
+            <version>3.0.1</version>
         </dependency>
 
         <dependency>
             <groupId>jakarta.enterprise</groupId>
             <artifactId>jakarta.enterprise.cdi-api</artifactId>
-            <version>3.0.0-M4</version>
+            <version>4.0.0.Alpha1</version>
```

Comment:
IIUC, CDI 4.0 is part of EE 10, but Jakarta REST 3.1 should depend on EE 9/9.1 technologies.  Should this stay at CDI 3.X?

Reply:
I am +-0 here, as this is just a dependency of some example, and nobody restricts examples to Java EE. What do others think?

Reply:
@andymc12  Replace 4.0.0.Alpha1 by 3.0.0 in https://github.com/eclipse-ee4j/jaxrs-api/pull/980/commits/7ff2b9ec95fe7bbc01e9a42fad2028a548b29d71. Fine?